services/Kraken.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Kraken",
+  "documents": {
+    "Terms of Service": {
+      "fetch": "https://www.kraken.com/en-us/legal#complete-tos",
+      "select": {
+        "startBefore": "#complete-tos",
+        "endBefore": "#margin-disclosure-statement"
+      }
```

Comment:
Thanks for submitting this PR 👍

To prevent this error `The document cannot be accessed or its content can not be selected: Received HTTP code 503 when trying to fetch 'https://www.kraken.com/en-us/legal/privacy'` you have to add `"executeClientScripts":true` for each documents declarations.
It should be fine after that.

Reply:
I now understood how to test my contributions ;)
Sorry if my commit is not perfectly done, I'll try to be better for the next PRs.
I requested a new review.

Reply:
your PR is much appreciated, thank you !